### PR TITLE
Fix old TNoodle links

### PR DESCRIPTION
**Pull request by campos20**

### CODE REVIEW

File: WcaOnRails/app/controllers/api/v0/api_controller.rb [hunk 1]
```diff
@@ -28,11 +28,11 @@ def auth_results
...
     render json: {
       "current" => {
-        "name" => "TNoodle-WCA-0.15.1",
+        "name" => "TNoodle-WCA-0.16.2",
         "information" => "#{root_url}regulations/scrambles/",
-        "download" => "#{root_url}regulations/scrambles/tnoodle/TNoodle-WCA-0.15.1.jar",
+        "download" => "#{root_url}regulations/scrambles/tnoodle/TNoodle-WCA-0.16.2.jar",
       },
-      "allowed" => ["TNoodle-WCA-0.15.1"],
+      "allowed" => ["TNoodle-WCA-0.15.1","TNoodle-WCA-0.16.2"],
```
Comment (viroulep) [inline WcaOnRails/app/controllers/api/v0/api_controller.rb]: Nit: travis is complaining that a space is missing after the coma.

**Comment by campos20** [comment]
Hopefully, branch's name is not a problem

**Comment by suushiemaniac** [comment]
Please refrain from merging this PR immediately, there is an open question I need to discuss with Alexandre first.

**Comment by viroulep** [comment]
Is there any more discussions pending?
The diff is quite different from the initial PR and as is it seems pretty straightforward to merge it up.

**Comment by viroulep** [comment]
The links are currently broken so I'm just going to merge that ^^

**Pull request merged**